src/services/ledger/index.types.d.ts

```diff
@@ -203,7 +203,7 @@ type IntraledgerLedgerMetadata =
 
 type SendLedgerMetadata = AddOnchainSendLedgerMetadata | AddLnSendLedgerMetadata
 
-type PaginationArgs = { after?: LedgerTransactionId; before?: LedgerTransactionId }
+type PaginatedArray<T> = { slice: T[]; total: number }
```

Comment:
Maybe let's come up with a better name for this. `slice` is kinda confusing because it's also a method on array.

Reply:
name looks ok to me. it's not exposed as an API or written in DB so would be straightforward to update if we wanted to